api/server/router/network/backend.go

```diff
@@ -11,7 +11,6 @@ import (
 // to provide network specific functionality.
 type Backend interface {
 	FindNetwork(idName string) (libnetwork.Network, error)
-	GetNetworkByName(idName string) (libnetwork.Network, error)
```

Comment:
On the same ground, should we also remove `GetNetworksByID` from this interface ?